projects/extension/sql/idempotent/013-vectorizer-api.sql

```diff
@@ -259,6 +286,9 @@ begin
       , 'formatting', formatting
       , 'scheduling', scheduling
       , 'processing', processing
+      , 'embedding_batch_schema', embedding_batch_schema
+      , 'embedding_batch_table', embedding_batch_table
+      , 'embedding_batch_chunks_table', embedding_batch_chunks_table
```

Comment:
We won't need this because they will be stored in the `embedding` variable.

Reply:
Done.